eth/backend.go

```diff
@@ -404,10 +405,38 @@ func New(stack *node.Node, config *ethconfig.Config, logger log.Logger) (*Ethere
 	if casted, ok := backend.engine.(*ethash.Ethash); ok {
 		ethashApi = casted.APIs(nil)[1].Service.(*ethash.API)
 	}
+	// proof-of-stake mining
+	validationPOS := func(random common.Hash, suggestedFeeRecipient common.Address, timestamp uint64) (*types.Block, error) {
```

Comment:
Better call it "assemblePayloadPOS" or something. "Validation" is confusing here.

Reply:
I think a good name would be "proposing" so we follow the same idea as the "mining" variable.